Source/Revit.IFC.Import/Data/IFCProduct.cs

```diff
@@ -264,6 +264,28 @@ protected override void Create(Document doc)
                else if (ObjectLocation != null)
                   lcs = lcs.Multiply(ObjectLocation.TotalTransform);
 
+               BasePoint basePoint = new FilteredElementCollector(doc).WherePasses(new ElementCategoryFilter(BuiltInCategory.OST_ProjectBasePoint)).First() as BasePoint;

```

Comment:
Can you add some comments for this code?  It would be easier to understand what it does.  I know that the current code isn't exactly teeming with comments, but if we are going to be on Git, we may as well improve practices there :)